### ISSUE TITLE
Compatibility with Front Matter Title Plugin

### ISSUE THREAD

**Issue by mangost23** [ozntel/file-tree-alternative#204]
Hi, the plugin does not work with the Front Matter Title Plugin. Please fix it.
The author of the plugin says that in order for it to work, you must add API support.
https://github.com/snezhig/front-matter-plugin-api-provider

**Comment by ggroh** [comment]
> Hi, the plugin does not work with the Front Matter Title Plugin. Please fix it. The author of the plugin says that in order for it to work, you must add API support. https://github.com/snezhig/front-matter-plugin-api-provider

I use zettelkasten IDs as file names. This feature is essential for me.!

**Comment by vesperizo** [comment]
Yes please, I would love to have the compatibility. It would be perfect!
